packages/dev/codemods/package.json

```diff
@@ -40,6 +40,10 @@
     "@types/jscodeshift": "^0.11.11",
     "typescript": "^5.5.0"
   },
+  "peerDependencies": {
+    "react": "^18.0.0 || ^19.0.0-rc.1",
```

Comment:
reduced set since s2 doesn't work with anything less, may as well require they update before they run the codemod